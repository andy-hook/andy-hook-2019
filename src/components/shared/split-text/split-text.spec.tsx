import React from "react"

import SplitText from "./split-text"
import { shallowWithTheme } from "@test-utils"

describe("<SplitText />", () => {
  it("renders correctly", () => {
    const tree = shallowWithTheme(
      "dark",
      <SplitText visible={true} animate={false}>
        This is test text
      </SplitText>
    )

    expect(tree).toMatchSnapshot()
  })
})
