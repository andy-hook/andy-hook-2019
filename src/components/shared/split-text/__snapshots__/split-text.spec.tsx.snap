// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SplitText /> renders correctly 1`] = `ShallowWrapper {}`;
